OKTA-620014 Updates to Web App with Redirect Quickstart (#4295)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app-redirect/main/spring-boot/getuserinfo.md

Since you requested the scopes `openid profile email`, Okta also returns an ID token along with the access token. You can parse out the claims in the ID token to find the user's profile information.

> For example, once the user has signed in, you can extract the user's name from the ID token and show it in the app:

Reviewer: ```suggestion
For example, after the user signs in, you can extract the user's name from the ID token and show it in the app:
```
